packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1101,6 +1101,15 @@ function commitPlacement(finishedWork: Fiber): void {
       // down its children. Instead, we'll get insertions from each child in
       // the portal directly.
     } else if (node.child !== null) {
+      if (
+        node.tag === SuspenseComponent &&
+        node.memoizedState === null &&
+        typeof node.memoizedProps.unstable_do_not_use_getDOMNodes === 'function'
+      ) {
+        node.memoizedProps.unstable_do_not_use_getDOMNodes(
+          getChildrenOfSuspenseBoundary(finishedWork),
+        );
+      }
```

Comment:
Need to pick off `unstable_do_not_use_getDOMNodes` before calling it, otherwise its `this` context will be the memoized props:

```js
const getDOMNodes = node.memoizedProps.unstable_do_not_use_getDOMNodes;
if (typeof getDOMNodes === 'function') {
  getDOMNodes(...)
}
```